packages/expo-splash-screen-command/src/configure.ts

```diff
@@ -0,0 +1,203 @@
+  }
+
+  return {
+    ...configuration,
+    backgroundColor: colorString.to.hex(backgroundColor.value),
+  };
+}
+
+async function runAsync() {
+  program
+    .arguments('<backgroundColor> [imagePath]')
+    .description(
+      'Idempotent operation that configures native splash screens using passed .png file that would be used in native splash screen.',
+      {
+        backgroundColor: `(${chalk.dim.red(
+          'required'
+        )}) Valid css-formatted color (hex (#RRGGBB[AA]), rgb[a], hsl[a], named color (https://drafts.csswg.org/css-color/#named-colors)) that would be used as background color for native splash screen view.`,
+        imagePath: `(${chalk.dim.yellow('optional')}) Path to a valid .png image.`,
+      }
+    )
+    .allowUnknownOption(false)
+    .option(
+      '-r, --resizeMode [resizeMode]',
+      `ResizeMode to be used for native splash screen image. Available values: ${getAvailableOptions(
+        ResizeMode
+      )} (${chalk.yellow.dim(`only available for ${chalk.cyan.dim('android')} platform)`)}).`,
+      (userInput: string) => {
+        if (!Object.values<string>(ResizeMode).includes(userInput)) {
+          console.log(
+            chalk.red(
+              `\nUnknown value ${chalk.yellow(userInput)} for option ${chalk.magenta(
+                'resizeMode'
+              )}. See below for the available values for this option.\n`
+            )
+          );
+          program.help();
+        }
+        return userInput;
+      },
+      ResizeMode.CONTAIN
+    )
+    .option(
+      '-p, --platform [platform]',
+      `Selected platform to configure. Available values: ${getAvailableOptions(Platform)}.`,
+      (userInput: string) => {
+        if (!Object.values<string>(Platform).includes(userInput)) {
+          console.log(
+            chalk.red(
+              `\nUnknown value ${chalk.yellow(userInput)} for option ${chalk.magenta(
+                'platform'
+              )}. See below for the available values for this option.\n`
+            )
+          );
+          program.help();
+        }
+        return userInput;
+      },
+      Platform.ALL
+    )
+    .asyncAction(
+      async (
+        backgroundColor: string,
+        imagePath: string | undefined,
+        { resizeMode, platform }: program.Command & Options
+      ) => {
+        const configuration = { imagePath, backgroundColor, resizeMode, platform };
+        const validatedConfiguration = await validateConfiguration(configuration);
+        await action(validatedConfiguration);
+      }
+    );
+
+  program.parse(process.argv);
+
+  // With no argument passed command should prompt user about wrong usage
+  if (program.args.length === 0) {
+    console.log(
+      chalk.red(
+        `\nMissing argument ${chalk.yellow.dim(
+          'backgroundColor'
+        )}. See below for the required arguments.\n`
+      )
+    );
+    program.help();
+  }
+}
+
+async function run() {
+  await runAsync().catch(e => {
```

Comment:
If you're using `async/await` then `try { ... } catch { ... }` is a preferred way to catch rejections